index.html

```diff
@@ -1,9 +1,78 @@
+                    <li><a href="#"> Resources </a></li>
+                    <li><a href="#"> Docs </a></li>
+                    <li><a href="#"> Support </a></li>
+                </ul>
+            </nav>
+        </div>
+        <div>
+            <img src="images/npm-logo.png" alt="npm logo" id="npm-logo">
+            <form action="search" id="search-bar">
+                <label for="search-text"><img src="images/magnifying-glass.png" alt="Search" id="search-icon"></label>    
+                <input type="search" name="search-text" id="search-text" placeholder="Search packages">
+                <button type="submit" id="search-button">Search</button>
+            </form>
+            <div id="user-access">
+                <a href="#" id="header-join">Join</a>
+                <a href="#" id="header-login">Log In</a>
+            </div>
+        </div>
+    </header>
+    <section id="join" class="column-section beige-bg">
+        <h1>Build amazing things</h1>
+        <article>Essential JavaScript development tools that help you go to market faster and build powerful applications using modern open source code.</article>
+        <div>
+            <a href="#" id="see-plans">See plans</a>
+            <a href="#" id="join-free">Join for free</a>
+        </div>
+    </section>
+    <section id="description" class="column-section">
+        <img src="images/triangle-hexagon.svg" alt="hexagon">
+        <h2>Bring the best of open source to your company</h2>
+        <article>npm is the tool used by over 11,000,000 JavaScript developers around the world. Your developers already use it. Your company depends on it. Create an Org and get more out of the tools your team already knows and loves.</article>
+    </section>
+    <section id="features" class="column-section">
+        <div id="features-container">
+            <div class="feature-box">
+                <img src="images/zero-configuration.svg" alt="">
+                <h4 class="feature-title">Zero configuration</h4>
+                <p class="feature-text">Create an org, add your team, and start collaborating. Nothing to configure, set up, or manage.</p>
+            </div>
+            <div class="feature-box">
+                <img src="images/team-management.svg" alt="">
+                <h4 class="feature-title">Team management</h4>
+                <p class="feature-text">Control who has access to what modules within your team namespace using straightforward team management capabilities.</p>
+            </div>
+            <div class="feature-box">
+                <img src="images/familiar-features.svg" alt="">
+                <h4 class="feature-title">Familiar features</h4>
+                <p class="feature-text">npm Orgs has 100% parity with all the public npm registry features your developers already use.</p>
+            </div>
+            <div class="feature-box">
+                <img src="/images/npm-audit.svg" alt="">
```

Comment:
Al tener el slash al principio del images no hace la referencia al archivo y al cargar tu pagina no aparece, podemos omitirlo y ya esta !!!

```suggestion
  <img src="images/npm-audit.svg" alt="">
```